aztk_sdk/utils/constants.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+# TODO: Separate aztk constants from software specific constants
+ 
+"""
+    DOCKER
+"""
+DEFAULT_DOCKER_REPO = "jiata/aztk:0.1.0-spark2.2.0-python3.5.4"
+DOCKER_SPARK_CONTAINER_NAME = "spark"
+DEFAULT_FLINK_DOCKER_REPO = "jafreck/aztk:flink"
+DOCKER_FLINK_CONTAINER_NAME = "flink"
+
+# DOCKER
+DOCKER_SPARK_WEB_UI_PORT = 8080
+DOCKER_SPARK_WORKER_UI_PORT = 8081
+DOCKER_SPARK_JUPYTER_PORT = 8888
+DOCKER_SPARK_JOB_UI_PORT = 4040
+DOCKER_SPARK_HOME = "/home/spark-current"
+DOCKER_FLINK_HOME = "/opt/flink"
```

Comment:
spark and flink should both be in `home` or `opt`. i think `opt` actually makes more sense / more common (which means that this fix is an item for me i guess)